vspace/src/main/java/edu/asu/diging/vspace/web/exhibit/publicpages/ExhibitionPublicAboutPageController.java

```diff
@@ -0,0 +1,29 @@
+package edu.asu.diging.vspace.web.exhibit.publicpages;
```

Comment:
I'm not a huge fan of the package names. I don't have a suggestion for better subpackage names than view and pages, but maybe at least drop the public part (since the `exhibit` part is intended to convey that)?
`edu.asu.diging.vspace.web.exhibit.pages`
`edu.asu.diging.vspace.web.exhibit.view`

Reply:
Changed